docs/content/en/docs/reference/changelog.md

```diff
@@ -11,6 +11,18 @@ menu:
 
 ### Added
 
+## [v0.9.0](https://github.com/aws/eks-anywhere/releases/tag/v0.9.0)
+
+### Added
+- Adding support to EKS Anywhere for a generic git provider as the source of truth for GitOps configuration management. [#9](https://github.com/aws/eks-anywhere/projects/9)
+- Allow users to configure Cloud Provider and CSI Driver with different credentials
+
+- Support to install, configure and maintain operational components that are secure and tested by Amazon on EKS Anywhere clusters.
+
+###Fixed
```

Comment:
Not sure whether it is a browser rendering issue but the ###Fixed is showing up as ###Fixed in preview.